src/actions/assetsActions.js

```diff
@@ -631,13 +631,30 @@ export const startAssetsSearchAction = () => ({
 
 export const searchAssetsAction = (query: string) => {
   return async (dispatch: Dispatch, getState: GetState, api: SDKWrapper) => {
+    const { assets: { supportedAssets } } = getState();
+    const search = query.toUpperCase();
+
+    const filteredAssets = supportedAssets.filter(({ name, symbol }) => {
+      return name.toUpperCase().includes(search) || symbol.toUpperCase().includes(search);
+    });
+
+    if (filteredAssets.length > 0) {
```

Comment:
We had some issues with assets list object before, I would switch it to `lodash.isempty`. However, the issue was due API `fetch` requester failing to produce timeout as exception and after a switch to `axios` this shouldn't happen anymore, but since we're putting this to reducer I would go with `lodash.isempty` here.

Reply:
`supportedAssets.filter()` always return an array, so we could be safe here

Reply:
The problem could be if the `supportedAssets` itself is not an array, but we should fix that in reducer then and keep the state consistent